### ISSUE TITLE
Check for gyb update causes exception

### ISSUE THREAD

**Issue by crcastle** [jay0lee/got-your-back#102]
Full steps to reproduce the issue:
1. Run `python3 gyb.py`
2. See exception and non-zero exit status
I've been using gyb for several months now within a script.  It's great!  Thanks for your work on it!  Found what I think is a bug though.

**Expected outcome (what are you trying to do?)**:
Update announcement should not cause exception (but getting notified that an update is available is great!), and program should continue to run normally.

**Actual outcome (what errors or bad behavior do you see instead?)**:
Here is the output I get:
```
👉  python3 got-your-back-0.45/gyb.py

There's a new version of GYB!!!

Traceback (most recent call last):
  File "got-your-back-0.45/gyb.py", line 1501, in <module>
    doGYBCheckForUpdates()
  File "got-your-back-0.45/gyb.py", line 325, in doGYBCheckForUpdates
    sys.stderr.write(announcement)
TypeError: write() argument must be str, not bytes
```
(non-zero exit status)

I get the same error in both 0.45 and 0.46.  Obviously I don't get it in 0.47 because that is the most recent version (as of creation of this issue).  Also I obviously don't get the error after running `touch noupdatecheck.txt` in the got-your-back-0.4x directory.

More info that might help in debugging:
```
👉  python3 --version
Python 3.5.1
```

```
👉  uname -a
Darwin pineapple.internal.acmeco.com 15.5.0 Darwin Kernel Version 15.5.0: Tue Apr 19 18:36:36 PDT 2016; root:xnu-3248.50.21~8/RELEASE_X86_64 x86_64
```

<img width="585" alt="screen shot 2016-05-29 at 6 23 47 pm" src="https://cloud.githubusercontent.com/assets/275734/15637333/6ae7cdf8-25ca-11e6-98af-1c0ffd05abfe.png">

**Comment by fosils** [comment]
I got the same error. Hoping for a fix soon. I still love GYB, though!

**Comment by jay0lee** [comment]
The fix is in 0.47. The workaround for older versions is to create a file called noupdatecheck.txt in the same folder as gyb.
